### PR TITLE
📐 Expose height css var

### LINKED ISSUE #39: Set specific height
There doesn't seem to be a way to set the height of the toolbar. It would be nice if there was a --paper-toolbar-height variable that we could set rather than using preset heights.

### PR DESCRIPTION
**Pull request by samccone**
Fixes #39

### CODE REVIEW

**Comment by chuckh** [comment]
LGTM

**Comment by blasten** [comment]
@samccone LGTM too. I only have two comments: 
1. Fix the tests: **has expected medium-tall height** and **has expected tall height**
2. Add the new variables to the readme.md as well.

**Comment by blasten** [comment]
friendly ping @samccone

**Comment by samccone** [comment]
oh yes will do today!

**Comment by samccone** [comment]
:pineapple: Emoji fun fact of the day:

> The pineapple made its way to England in the 17th century and by the 18th century, being seen with one was an instant indicator of wealth -- a single pineapple could cost the equivalent of $8,000 today. In fact, the fruit was so desirable and rare that consumers often rented a pineapple for the night to show off to fellow party-goers.

File: paper-toolbar.html [hunk 1]
```diff
@@ -102,11 +104,11 @@
     }
 
     :host(.medium-tall) {
-      height: 128px;
+      height: var(--paper-toolbar-height, 128px);
```
Comment (samccone) [inline paper-toolbar.html]: These changes are to work around
https://github.com/Polymer/polymer/issues/2531 :bug:

Comment (blasten) [inline paper-toolbar.html]: @samccone what about if we use `calc(var(--paper-toolbar-height, 64px) * 2);`  and `calc(var(--paper-toolbar-height, 64px) * 3);` for `.tall`  Alternatively, we could have different custom properties for those classes.  wdyt? See this request: https://github.com/PolymerElements/paper-toolbar/issues/6

Comment (samccone) [inline paper-toolbar.html]: sure I like the \* approach, will update

**Comment by samccone** [comment]
@blasten let me know what you think :thought_balloon:

**Comment by clintwood** [comment]
@samccone, @blasten, this doesn't make sense without also applying `--paper-toolbar-height` to `.toolbar-tools`, [here](https://github.com/samccone/paper-toolbar/blob/master/paper-toolbar.html#L116) and [here](https://github.com/samccone/paper-toolbar/blob/master/paper-toolbar.html#L139) as mentioned in #44!

**Comment by samccone** [comment]
great good call @clintwood I will add this to the PR

**Comment by clintwood** [comment]
Awesome!

**Comment by blasten** [comment]
+1 `--paper-toolbar-sm-height`

**Comment by samccone** [comment]
@blasten can you explain what you mean?

**Comment by samccone** [comment]
also @blasten @clintwood this should be all set now

**Comment by samccone** [comment]
@blasten :)

**Comment by blasten** [comment]
LGTM. merged now, thanks!

**Comment by samccone** [comment]
:bow:

**Comment by clintwood** [comment]
Great stuff!

**Pull request merged**